portfolio/src/main/webapp/script.js

```diff
@@ -37,3 +38,20 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function getServerName(){
+    fetch('/data').then(response => response.json()).then((stats) => {
+        const nameContainer = document.getElementById('name-container');
+        nameContainer.innerHTML="";
+        for(let i=0; i<stats.length; i++){
+            nameContainer.appendChild(
+            createListElement(stats[i]));
+        }
+    });
+}
+
+function createListElement(text){
+    const liElement = document.createElement('li');
+    liElement.innerText = text;
+    return liElement;
+}
```

Comment:
We should always add a newline to the end of our files. https://en.wikipedia.org/wiki/Newline